prow/plugins/verify-owners/verify-owners.go

```diff
@@ -231,16 +231,18 @@ func handle(ghc githubClient, gc git.ClientFactory, roc repoownersClient, log *l
 			log.WithError(err).Error("Error cleaning up repo.")
 		}
 	}()
-	if err := r.CheckoutPullRequest(number); err != nil {
+	if err := r.Config("user.name", "prow"); err != nil {
 		return err
 	}
-	// If we have a specific SHA, use it.
-	if pr.Head.SHA != "" {
-		if err := r.Checkout(pr.Head.SHA); err != nil {
-			return err
-		}
+	if err := r.Config("user.email", "prow@localhost"); err != nil {
+		return err
+	}
+	if err := r.Config("commit.gpgsign", "false"); err != nil {
+		log.WithError(err).Errorf("Cannot set gpgsign=false in gitconfig: %v", err)
```

Comment:
Nit: The error in the `Errorf` statement is redundant, it is already logged in the `WithError` statement